### ISSUE TITLE
nextcloudcron.service file fails produces error on reboot.

### ISSUE THREAD

**Issue by Steve8291** [nextcloud/documentation#1584]
After installing `nextcloudcron.service` and `nextcloudcron.timer` and enabling them in systemd they create an error on reboot. It appears that `nextcloudcron.service` is starting up before `mysql.service`  
I've added the line `After=mysql.service` to the service file. Would it be prudent to make this change in the documentation?
Referencing this file: [https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/background_jobs_configuration.rst](https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/background_jobs_configuration.rst)
```
Description=Nextcloud cron.php job
After=mysql.service

[Service]
User=www-data
ExecStart=/usr/bin/php -f /var/www/nextcloud/cron.php

[Install]
WantedBy=basic.target
```
Also noticed what appears to be a typo:  
`When the option --now is used with enable, the resp. unit will also be started.`  
Is "resp." supposed to read `.service` ?

**Comment by skjnldsv** [comment]
@Steve8291 thanks for the suggestion :)
If you think it make sense, please provide a patch for the documentation! :rocket: 

I think it does make sense to wait for the dependencies services to be ready

**Comment by kesselb** [comment]
> When the option --now is used with enable, the resp. unit will also be started.
Is "resp." supposed to read .service

Unit is  the official naming: https://www.freedesktop.org/software/systemd/man/systemd.unit.html

**Comment by Steve8291** [comment]
Ah. "resp." is the abbreviation for "respective". I get it now. Sorry. I guess I had never seen it abbreviated before.

**Comment by kesselb** [comment]
I think this has been fixed by #1603
